lib/opal/nodes/def.rb

```diff
@@ -52,9 +52,14 @@ def compile
 
           add_temp 'self = this'
 
-          compile_rest_arg
-          compile_opt_args
-          compile_keyword_args
+          if has_complex_args?
+            compile_complex_args(false)
```

Comment:
maybe it's better to use `"false"` instead of `false` since it's gonna be JavaScript… in addition it might be better to clarify what's that using kwargs: `compile_complex_args(expand_code: 'false')` or something like that